tests/contrib/django/app/views.py

```diff
@@ -23,10 +26,34 @@ def get(self, request, *args, **kwargs):
         return HttpResponse(status=403)
 
 
+def function_view(request):
+    return HttpResponse(status=200)
+
+
+class FeedView(Feed):
```

Comment:
OK, that was the missing bit I couldn't figure out not knowing Django enough I guess. Thanks!